### PR TITLE
Update mediawiki wikifarm lists

### PR DESCRIPTION
**Pull request by simonliu99**
continuing to update lists and spiders

edit: you can leave this PR open until i finish up the spiders

### CODE REVIEW

File: listsofwikis/mediawiki/shoutwiki-spider.py [hunk 1]
```diff
@@ -1,55 +1,73 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-# Copyright (C) 2014 WikiTeam developers
+# Copyright (C) 2021 Simon Liu
```
Comment (nemobis) [inline listsofwikis/mediawiki/shoutwiki-spider.py:4]: Doesn't matter much, but: 2021?

Comment (simonliu99) [reply listsofwikis/mediawiki/shoutwiki-spider.py:4]: oops i guess i'm still mentally living in 2021 🥲

**Comment by simonliu99** [comment]
good to merge now

**Comment by nemobis** [comment]
Thanks!

**Pull request merged**